specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-10-02-preview/managedClusters.json

```
@@ -5113,6 +5113,39 @@
           },
           "title": "The upgrade channel for auto upgrade. The default is 'none'.",
           "description": "For more information see [setting the AKS cluster auto-upgrade channel](https://docs.microsoft.com/azure/aks/upgrade-cluster#set-auto-upgrade-channel)."
+        },
+        "nodeOSUpgradeChannel": {
+          "type": "string",
+          "enum": [
+            "Unmanaged",
+            "None",
+            "SecurityPatch",
+            "NodeImage",
+          ],
+          "x-ms-enum": {
+            "name": "nodeOSUpgradeChannel",
```

minor:
```suggestion
            "name": "NodeOSUpgradeChannel",
```

Practically this doesnt matter because all the SDKs will generate the type with the uppercase first letter anyway, but stylistically I think this is slightly better